gen/definitions/vlan_tag.yaml

```diff
@@ -22,12 +22,14 @@ attributes:
     type: String
     value: "VlanTag"
   - model_name: startTag
+    tf_name: start_tag
```

Comment:
I think the `tf_name` here is redundant, as by default the `model_name` gets translated to a `tf_name` of `start_tag`.

Reply:
done